README.md

```diff
@@ -5,89 +5,122 @@
 
-### Presenting About Pi-hole
+### Presentations about Pi-hole
+Word-of-mouth continues to help our project grow immensely, and we'd like to help those who are going to be presenting Pi-hole at a conference, meetup or even a school project. If you'd like some free swag to hand out to your audience, [get in touch with us](https://pi-hole.net/2017/05/17/giving-a-presentation-on-pi-hole-contact-us-first-for-some-goodies-and-support/).
 
-Word-of-mouth has immensely helped our project grow.  If you are going to be presenting about Pi-hole at a conference, meetup, or even for a school project, [get a hold of us for some free swag](https://pi-hole.net/2017/05/17/giving-a-presentation-on-pi-hole-contact-us-first-for-some-goodies-and-support/) to hand out to your audience.
+-----
 
-# Overview Of Features
+## Getting in touch with us
+- [Users Forum](https://discourse.pi-hole.net/)
+- [Feature requests](https://discourse.pi-hole.net/c/feature-requests?order=votes)
+- [FAQs](https://discourse.pi-hole.net/c/faqs)
+- [Wiki](https://github.com/pi-hole/pi-hole/wiki)
+- [/r/pihole on Reddit](https://www.reddit.com/r/pihole/)
+- [@The_Pi_Hole on Twitter](https://twitter.com/The_Pi_Hole)
+- [Pi-hole on YouTube](https://www.youtube.com/channel/UCT5kq9w0wSjogzJb81C9U0w)
+- [ThePiHole on Facebook](https://www.facebook.com/ThePiHole/)
+- [Chat on Gitter](https://gitter.im/pi-hole/pi-hole?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-## The Dashboard (Web Interface)
+-----
 
-The [dashboard](https://github.com/pi-hole/AdminLTE#pi-hole-admin-dashboard) will (by default) be enabled during installation so you can view stats, change settings, and configure your Pi-hole.
+## Features
+* <sub>[The Web Admin Dashboard](#the-web-admin-dashboard)</sub>
+* <sub>[The Faster-Than-Light Engine](#the-faster-than-light-engine)</sub>
+* <sub>[The Query Log](#the-query-log)</sub>
+* <sub>[Long-term Statistics](#long-term-statistics)</sub>
+* <sub>[Whitelisting and Blacklisting](#whitelisting-and-blacklisting)</sub>
+* <sub>[Additional Blocklists](#additional-blocklists)</sub>
+* <sub>[Enable and Disable Pi-hole](#enable-and-disable-pi-hole)</sub>
+* <sub>[Tools](#tools)</sub>
+* <sub>[Web Admin Settings](#web-admin-settings)</sub>
+* <sub>[Built-in DHCP Server](#built-in-dhcp-server)</sub>
+* <sub>[Real-time Statistics](#real-time-statistics)</sub>
+
+### The Web Admin Dashboard
+The optional dashboard (based on [almasaeed2010's AdminLTE](https://github.com/almasaeed2010/AdminLTE) Admin Control Panel) allows you to view stats, change settings, and configure your Pi-hole.
```

Comment:
Since we didn't fork the repo on GitHub, I feel it'd be good of us to have *some* nod to the source, but I'm totally up for discussion on this :smile: